x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/source_data.tsx

```diff
@@ -687,8 +667,8 @@ export const staticSourceData = [
       'xpack.enterpriseSearch.workplaceSearch.sources.sourceDescriptions.slack',
       {
         defaultMessage:
-          '{sourceName} is a communication tool that enables real-time collaboration and decision making. With ${sourceNames.slack}, keep track of the work happening across teams, engage directly with your coworkers on ongoing projects and communicate with other organizations.',
```

Comment:
This was caught by the build. ` ${sourceNames.slack}` should have been `{sourceName}`

Reply:
[Context](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/88117/execution/node/369/log/)